source/includes/rest/_subscribers.md

```diff
@@ -117,6 +119,14 @@ On January 7, 2019 we renamed Subscribers to People to better represent who you
       <td><code>phone</code></td>
       <td>The subscriber's primary phone number.</td>
     </tr>
+    <tr>
+      <td><code>sms_number</code></td>
+      <td>Optional. String. The subscriber's mobile phone number in E.164 formatting. E.g. <code>"+16125551212"</code>. Only US-based numbers are supported at this time.</td>
```

Comment:
Do we want to say explicitly that this will return a validation error?

Reply:
I'm torn. I like the idea. But it doesn't seem like we do that for the other attributes. We have this catch-all errors page:
https://developer.drip.com/#validation-errors

But I also just noticed something. We need to add `sms_error` to that list. Example response with a bad number:

```json
{
  "errors": [
    {
      "code": "sms_error",
      "attribute": "sms_number",
      "message": "SMS number SMS number must be from an SMS supported country"
    },
    {
      "code": "sms_error",
      "attribute": "sms_number",
      "message": "SMS number SMS number is not valid mobile number"
    }
  ]
}

And I see what we're duplicating "SMS number" in the message.

I'll take care of these two issues.

Reply:
Probably best to be consistent for now